adapters/onebot/src/types.ts

```diff
@@ -89,6 +89,11 @@ export interface UnidirectionalFriendInfo extends AccountInfo {
   source: string
 }
 
+export interface Author extends Universal.Author {
+  time?: string | number
+  messageId?: string
```

Comment:
What's the use of this `messageId`?

Reply:
If you are to send forward message with id, you should use `<message id>` rather than `<author message-id>`.

Reply:
> What's the use of this `messageId`?

From go-cqhttp document:
![image](https://user-images.githubusercontent.com/64234553/218254085-4443a84c-0403-4928-977c-e92246904991.png)

> If you are to send forward message with id, you should use `<message id>` rather than `<author message-id>`.

Yes, that's what I did.